_docs/_latest/configure/index.md

```diff
@@ -12,27 +12,36 @@ use these guides to configure features.
-| Configure Inventory to collect and store information about your GCP resources. Inventory helps you undersand your resources and take action to conserve resources, reduce cost, and minimize security exposure. |
+Configure Forseti global and module-specific settings by updating the centrally-maintained
+configuration file. This includes basic configuration, and configuration for Inventory, Scanner,
+and Enforcer.
 
-| **[Configuring Scanner]({% link _docs/latest/configure/scanner/index.md %})** |
-| :---------------------------------------------------------------------------- |
-| Configure Scanner to monitor your GCP resources for rule violations. Scanner uses the information from Inventory to regularly compare role-based access policies for your resources. |
+**[Configuring Inventory]({% link _docs/latest/configure/inventory/index.md %})**
 
-| **[Configuring Enforcer]({% link _docs/latest/configure/enforcer/index.md %})** |
-| :---------------------------------------------------------------------------- |
-| Configure Enforcer to automatically correct policy discrepancies. Enforcer uses policies you create to compare the current state of your Compute Engine firewall to the desired state and uses Google Cloud APIs to make changes if it finds any differences. |
+Configure Inventory to collect and store information about your GCP resources. Inventory helps you
+understand your resources and take action to conserve resources, reduce cost, and minimize security
+exposure.
 
-| **[Enabling Email Notifications]({% link _docs/latest/configure/email-notification.md %})** |
-| :---------------------------------------------------------------------------- |
-| Enable Forseti email notifications using the SendGrid API. SendGrid is the suggested free email service provider for Google Cloud Platform (GCP). |
+**[Configuring Scanner]({% link _docs/latest/configure/scanner/index.md %})**
 
-| **[Enabling G Suite Google Groups Collection]({% link _docs/latest/configure/gsuite-group-collection.md %})** |
-| :---------------------------------------------------------------------------- |
-| Enable the data collection of G Suite Google Groups for processing by Forseti Inventory. G Suite Groups Collection helps you make sure the right people are in the right group, and is required for Explain. |
+Configure Scanner to monitor your GCP resources for rule violations. Scanner uses the information
+from Inventory to regularly compare role-based access policies for your resources.
 
+**[Configuring Enforcer]({% link _docs/latest/configure/enforcer/index.md %})**
+
+Configure Enforcer to automatically correct policy discrepancies. Enforcer uses policies you
+create to compare the current state of your Compute Engine firewall to the desired state and uses
+Google Cloud APIs to make changes if it finds any differences.
+
+**[Enabling Email Notifications]({% link _docs/latest/configure/email-notification.md %})**
```

Comment:
Instead of just talking about email notifications here, maybe we should have a [Configuring Notifier] section instead?

Reply:
Sure, let's safe that for the reviewer of the configure section?